### PR TITLE
desktop-version design

### PR DESCRIPTION
**Pull request by Obimbo07**
This branch introduces the following:

1. The desktop version of our portfolio
2. linters fix 
3. media images of the desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -459,3 +459,177 @@ form {
   width: 134px;
   margin-top: 32px;
 }
+
+@media (min-width: 768px) {
```
Comment (OyePriscilla) [inline style.css:496]: - [ ] Good job done so far! 👏 👏 Kindly ensure that all the element in the body element are properly marginalised in accordance to the project requirement. 👍 

## Figma design
Untitled

- [ ] At exactly `768px`, all the details are not showing as expected; this means that your app is not yet fully responsive. Kindly don't forget to remove the horizontal scroll bar for good user experience. ℹ️ 

Untitled

File: style.css [hunk 2]
```diff
@@ -459,3 +459,177 @@ form {
...
+    color: #091e42;
+  }
+
+  .cards-box {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    gap: 30px;
+    border: 0 solid #ccc;
+  }
```
Comment (OyePriscilla) [inline style.css]: - [ ] Kindly ensure that the see project button pops up whenever a user hover on any of the projects. 🙏 

## Your design
Untitled

File: style.css [hunk 3]
```diff
@@ -459,3 +459,177 @@ form {
   width: 134px;
   margin-top: 32px;
 }
+
+@media (min-width: 768px) {
+  .hamberger-btn {
+    display: none;
+  }
```
Comment (OyePriscilla) [inline style.css]: - [ ] In accordance to best practises, kindly avoid code repetition. Hint: You repeated this line of code in line 577 and 632. ℹ️

File: index.html [hunk 4]
```diff
@@ -17,6 +17,12 @@
                 <label class="hamberger-btn" >
                 <img class="hamberger" id="ham" src="img/IconMenu.png">
                 </label>
+                <ul class="nav-links">
+                  <li><a href="#"></a>Portfolio</li>
+                  <li><a href="#"></a>About</li>
+                  <li><a href="#"></a>Contact</li>
+                  <a><img src="img/Union.png"><a>
+                </ul>
```
Comment (OyePriscilla) [inline index.html]: - [ ] Kindly resolve the interference of the destop nav-items in the mobile view, I believe you suppose to hide them. ℹ️ 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Obimbo07 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used` media queries `for the Desktop breakpoint in the CSS file ✔️ 
Used both `Flexbox and Grid` in order to layout the elements on the page ✔️ 

## Required Changes ♻️

- [ ] @Obimbo07  Kindly fix the linkedln broken link in the Author section of the README file, this will make your repo to be more engaging. 👍

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -255,6 +259,8 @@ header {
   border: 0;
   height: 48px;
   background: #ff6b00;
+  transition: background-color 0.5s ease;
+  display: none;
```
Comment (wayungi) [inline style.css]: - [ ] As previously pointed out by the last reviewer,  Kindly write DRY CSS.  I counted 5 instances of the `display: none` rule. Please create a hidden class with the display none property and assign it to all elements you do not want to display 😓 
```
.hidden  {
  display: none;
}
```

- [ ] Please  Kindly apply this comcept to all other instances of code duplication in your CSS file 🤝

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Obimbo07 , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Mobile version looks great ✔️ 
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] As requested by the previous reviewer, Kindly fix the LinkedIn broken link. Let it link to your linkedIn profile 😄 

- [ ] Please note that your elements are not aligned according to the Figma design. Kindly look at the comparisons below.

**Figma**
figma

**Your implemntation**
impl

**Required actions**
- [ ] The social media icons should be moved more towards the left edge as in the design 👍🏾 
- [ ] Align the projects to match the text in the header section. 👍🏾 
- [ ] The Title **My Recent works** should also be aligned with the texts in the header section 👍🏾 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Obimbo07** [comment]
Please consider my review i will work on the display none to make my CSS DRY.

**Review by Kamba56 — APPROVED** [review]
Hi @Obimbo07, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Great job using flexbox and grid to layout contents!👏👏👏
- [x] Great job building desktop version of portfolio!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**